appdata.patch

```diff
@@ -0,0 +1,12 @@
+diff --git program/org.kde.okteta.appdata.xml program/org.kde.okteta.appdata.xml
+index beed216b..0347c3aa 100644
+--- a/program/org.kde.okteta.appdata.xml
++++ b/program/org.kde.okteta.appdata.xml
+@@ -553,7 +553,6 @@
+     </screenshot>
+   </screenshots>
+   <project_group>KDE</project_group>
+-  <icon type="stock">okteta</icon>
```

Comment:
It would be nice to send this patch upstream if possible

Reply:
In progess: https://invent.kde.org/utilities/okteta/-/merge_requests/3 :slightly_smiling_face: